frontend/src/metabase/query_builder/components/NativeQueryEditor/AceEditor.module.css

```diff
@@ -0,0 +1,112 @@
+:global {
```

Comment:
This is turns all the classes in the css file "global" (ie. they don't get mangled by the css module preporcessor), except for the ones marked by `:local`.